src/components/Identity/userIds/createUserIds.js

```diff
@@ -69,5 +69,5 @@ export default ({
     }
   };
 
-  return customerIds;
+  return userIds;
```

Comment:
I personally find that having this module named `customerIds`/`userIds` is confusing, but I didn't want to get caught up in debates so I didn't attempt to change the pattern for this PR.